microhab/sample-with-examples.bib

```diff
@@ -0,0 +1,195 @@
+@manual{lamp87,
```

Comment:
you really didnt have any papers or projects to cite? not even past spex projects?